consai_robot_controller/src/field_info_parser.cpp

```diff
@@ -725,97 +725,120 @@ bool FieldInfoParser::avoid_obstacles(
 
   const double VISIBILITY_THRESHOLD = 0.01;  // 0.0 ~ 1.0
   // 自身から直進方向に何m離れたロボットを障害物と判定するか
-  const double OBSTACLE_DETECTION_X = 0.2;
+  const double OBSTACLE_DETECTION_X = 0.5;
   // 自身から直進方向に対して左右何m離れたロボットを障害物と判定するか
   const double OBSTACLE_DETECTION_Y = 0.5;
   // 相対的な回避位置
-  const double AVOIDANCE_POS_X = 0.2;
-  const double AVOIDANCE_POS_Y = 0.4;
+  double avoidance_pos_x = 0.0;
+  double avoidance_pos_y = 0.0;
+
+  // ボール回避判定フラグ
+  bool is_avoid = false;
```

Comment:
need_avoidance はどうでしょうか？

```suggestion
  bool need_avoidance = false;
```

Reply:
メモ：書いた